codecov.yml

```diff
@@ -1,7 +1,3 @@
-codecov:
-  bot: mbtace
-  notify:
-    require_ci_to_pass: true
```

Comment:
This option seems useful in its own right, or do we have to remove it if we want to use the non-bot approach? Would deleting only line 2 fix the issue?

Reply:
It's the default setting: https://docs.codecov.io/docs/codecov-yaml#section-default-yaml

I'll admit that I mostly only removed it to trigger a new build. But generally I prefer not to declare settings in configuration if they are the default value. If a config only reflects the non-default values, it's easier to read and understand.

Reply:
Plus in our case it was nested under the `notify` section (perhaps incorrectly?), so I'm not sure it was being used anyway.